.github/workflows/deploy-book.yml

```diff
@@ -54,3 +54,18 @@ jobs:
       - name: Deploy book
         if: github.event_name == 'push' && steps.branch.outputs.branch != null && github.repository == 'crystal-lang/crystal-book'
         run: aws s3 sync ./site s3://crystal-book/reference/${{ steps.branch.outputs.branch }} --delete
+
+  lint:
+    name: Check style issues
+    runs-on: ubuntu-latest
```

Comment:
Please pin a specific release. Otherwise we might see unexpected failures when the base image implicitly updates and breaks something.

Reply:
Hmm but we have `ubuntu-latest` in all workflows in this repo

Reply:
Okay, then we can leave it at that for this PR.
This is just something I've been alerted to because we've had some trouble in the past. We should change it in all our CI workflows.